examples/restore_model_dependencies/restore_model_dependencies_example.ipynb

```diff
@@ -0,0 +1,96 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70a2b95e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "from sklearn import datasets\n",
+    "from sklearn.neighbors import KNeighborsClassifier\n",
+    "from pathlib import Path\n",
+    "\n",
+    "X, y = datasets.load_iris(as_frame=True, return_X_y=True)\n",
+    "model = KNeighborsClassifier()\n",
+    "model.fit(X, y)\n",
+    "\n",
+    "model_path = \"/tmp/sk_model_01\"\n",
+    "\n",
+    "mlflow.sklearn.save_model(model, model_path)\n",
+    "\n",
+    "req_file_path = mlflow.pyfunc.get_model_dependencies(model_path)\n",
+    "\n",
+    "print(Path(req_file_path).read_text())"
```

Comment:
Can we run this line in a separate cell because `mlflow.pyfunc.get_model_dependencies` also prints out something?